lib/services/init-service.ts

```diff
@@ -56,6 +57,12 @@ export class InitService implements IInitService {
 					});
 				}
 
+				let dependencies = projectData["dependencies"];
+				if(!dependencies) {
+					projectData["dependencies"] = Object.create(null);
```

Comment:
Perhaps `projectData.dependencies = {}`? You can use {} here, because we just serialize the object immediately.